sdk/cosmos/sdk/pom.xml

```diff
@@ -0,0 +1,280 @@
+<!--
```

Comment:
Folder should be `sdk/cosmos/microsoft-azure-cosmos`

Reply:
will cover it in follow up PR too.